### PR TITLE
Hometask12

### PR DESCRIPTION
**Pull request by bogdanbasii**

### CODE REVIEW

File: Task1 [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    rating INTEGER DEFAULT 5
+);
+
+CREATE TABLE IF NOT EXISTS books (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    title TEXT,
+    author TEXT,
+    year INTEGER,
+    price INTEGER,
+    publishing_house_id INTEGER,
```
Comment (klymenok) [inline Task1]: It is better to add NOT NULL constraint for a field which is going to become a FOREIGN KEY

File: Task1 [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    FOREIGN KEY (publishing_house_id) REFERENCES publishing_house(id)
+);
+
+CREATE TABLE IF NOT EXISTS purchases (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    user_id INTEGER,
```
Comment (klymenok) [inline Task1]: It is better to add NOT NULL constraint for a field that is going to become a FOREIGN KEY (user_id and book_id)

**Review by klymenok — CHANGES_REQUESTED** [review]

File: Task1 (hunk 2 above)
Comment (bogdanbasii) [reply Task1]: Changes done

File: Task1 (hunk 1 above)
Comment (bogdanbasii) [reply Task1]: Changes done

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**